de/api/configuration-generate.md

```diff
@@ -172,28 +162,24 @@ const axios = require('axios')
 
 module.exports = {
   generate: {
-    routes: function () {
-      return axios.get('https://my-api/users')
-      .then((res) => {
-        return res.data.map((user) => {
-          return {
-            route: '/users/' + user.id,
-            payload: user
-          }
-        })
-      })
-    }
+    routes: () =>
+      axios.get('https://my-api/users').then(res =>
+        res.data.map(user => ({
+          route: '/users/' + user.id,
+          payload: user
+        }))
+      )
   }
 }
 ```
 
 Now we can access the `payload` from `/users/_id.vue` like so:
 
 ```js
-async asyncData ({ params, error, payload }) {
-  if (payload) return { user: payload }
-  else return { user: await backend.fetchUser(params.id) }
-}
+asyncData: async ({ params, error, payload }) =>
```

Comment:
While it might be optimised, I don't think it's a good idea for documentation since it makes it harder to read for a human :)

Reply:
Would like @manniL opinion on this particular comment

Reply:
@manniL it has come a full circle UwU

Reply:
I'd go for something like

```js
async asyncData ({ params, error, payload }) {
  if (payload) {
    return { user: payload }
  }
  const user = await backend.fetchUser(params.id)
  return { user }
}
```

Not all readers know ES6 and short circuits very well, so I'd favor a "more readable" /simple version

Reply:
Oke, will do.
How do you feel about the original version:
```js
async asyncData ({ params, error, payload }) {
  if (payload) return { user: payload }
  else return { user: await backend.fetchUser(params.id) }
}
```
i'd just change it to arrow notation since `this` isn't in use, and remove the unnecessary `else`
```js
asyncData: async ({ params, error, payload }) => {
  if (payload) return { user: payload }
  return { user: await backend.fetchUser(params.id) }
}
```

Reply:
@qm3ster I don't like "inline ifs" actually. Was a fan of them years ago but now I appreciate the extra braces for readability.

This is also aligns with our eslint config.

Reply:
Gotcha, will do like yours. Should the `await` also be on a separate line?
**whispers**: *multiline ifs are a code smell, pull them out into a named function*